src/token/token_service_repo.ml

```diff
@@ -50,13 +50,34 @@ struct
           $6,
           $7
         )
-|sql}
+        |sql}
 
     let insert connection ~token =
       let module Connection = (val connection : Caqti_lwt.CONNECTION) in
       Connection.exec insert_request token
       |> Lwt_result.map_err Caqti_error.show
 
+    let update_request =
+      Caqti_request.exec Model.t
+        (* TODO Check if correct sql syntax*)
+        {sql|
+        UPDATE token_tokens
+        SET
+          uuid = $1,
```

Comment:
@aronerben Are you sure you want to allow changing stuff like token_value and uuid?

Reply:
I removed those two, should I also remove `created_at`? Do we need an `updated_at` column?

Reply:
@aronerben Too early to decide, I'd say just do as the other repos atm.